examples/wrapper-components/react-javascript/src/App.js

```diff
@@ -1,74 +1,10 @@
 import './App.css';
-import Alert from './components/Alert.vue'
```

Comment:
There are a bunch of errors because the imports are wrong. They are .vue imports => Please test next time before committing.